### PR TITLE
Add learning center walkthroughs

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Add walkthroughs for Docker Desktop Learning Center in Guides.

### Related issues (optional)
DA-427

### CODE REVIEW

File: get-started/what-is-a-container.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+## Dive deeper
+
+### Breaking down the `docker run` command
+
+When setting up the container, you used the `docker run` command. Here are what the different parts of the command do:
+
+ - `docker run`: This is used to run containers. It needs at least one argument, and that argument is the image you want to run. In this case, it's `docker/welcome-to-docker`.
+ - `-p 8088:80`: This lets Docker know that port 80 in the container needs to be accessible from port 8088 on your local host.
+ - `-d`: This runs the container detached or in the background.
+ - `—-name welcome-to-docker`: This sets the name for your container. If you don’t do so, Docker will select a random name for you.
```
Comment (aevesdocker) [inline get-started/what-is-a-container.md:71]:
```suggestion
 - `—-name welcome-to-docker`: This sets the name for your container. If you don’t do so, Docker selects a random name for you.
```

File: get-started/what-is-a-container.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Step 4: Stop your container
+
+The `welcome-to-docker` container will continue to run until you stop it. To stop the container in Docker Desktop, go to the **Containers** tab and select the **Stop** icon in the **Actions** column of your container.
+
+![Stopping a container in Docker Desktop](images/getting-started-stop.png){:width="500px"}
+
+## What's next
+
+In this guide, you explored what a container is. Next, you'll learn what you need to create your own container.
```
Comment (aevesdocker) [inline get-started/what-is-a-container.md:45]:
```suggestion
In this guide, you explored what a container is. Next, Learn what you need to create your own container.
```

File: get-started/what-is-a-container.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                <a href="/get-started/run-your-own-container/"><img src="/assets/images/build-frontends.svg" alt="run your own container" width="70" height="70"></a>
+            </div>
+                <h3 id="run-your-own-container"><a href="/get-started/run-your-own-container">Create and run your own container</a></h3>
+            </div>
+        </div>
+  </div>  
+</div>
+
+
+## Dive deeper
```
Comment (aevesdocker) [inline get-started/what-is-a-container.md:62]: Nice. I like this at the end 🙌

File: get-started/run-your-own-container.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
+---
+title: Create and run your own container
+keywords: get started, quick start, intro, concepts
+description: Learn how to run a container from scratch
+---
+
+In this guide you'll learn the basic steps to run a container from scratch. For this guide, you'll use a sample Node.js application, but it's not necessary to know Node.js.
```
Comment (aevesdocker) [inline get-started/run-your-own-container.md:7]:
```suggestion
Follow this guide to learn the basic steps of running a container from scratch. This guide uses a sample Node.js application, but it's not necessary to know Node.js.
```

File: get-started/run-docker-hub-images.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+title: Run Docker Hub images
+keywords: get started, quick start, intro, concepts
+description: Learn how to run Docker Hub images
+---
+
+You can share and store images in Docker Hub ([http://hub.docker.com](http://hub.docker.com)). Docker hub has over 100,000 images created by developers that you can run locally. You can search for Docker Hub images and run them directly from Docker Desktop.
+
+## Step 1: Search for the image
+
+You can search for Docker Hub images on Docker Desktop. Use the search bar, or use the shortcut ⌘K on Mac or Ctrl + K on Windows. In the search, specify `welcome-to-docker` to find the `docker/welcome-to-docker` image you'll use in this guide.
```
Comment (aevesdocker) [inline get-started/run-docker-hub-images.md:11]:
```suggestion
You can search for Docker Hub images on Docker Desktop. Use the search bar, or use the shortcut ⌘K on Mac or Ctrl + K on Windows. In the search bar, specify `welcome-to-docker` to find the `docker/welcome-to-docker` image used in this guide.
```

File: get-started/run-docker-hub-images.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Step 3: Explore the container
+
+That's it! The container is ready to use. Go to the **Containers** tab in Docker Desktop to view the container.
+
+![Viewing the Containers tab in Docker Desktop](images/getting-started-view.png){:width="500px"}
+
+## What's next
+
+In this guide, your ran someone else's image from Docker Hub. Next, you'll learn how you can publish your own image to Docker Hub.
```
Comment (aevesdocker) [inline get-started/run-docker-hub-images.md:33]:
```suggestion
In this guide, you ran someone else's image from Docker Hub. Next, learn how you can publish your own image to Docker Hub.
```

File: get-started/publish-your-own-image.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+title: Publish your own image
+keywords: get started, quick start, intro, concepts
+description: Learn how to publish your own images
+---
+
+In this guide, you'll learn how you can share your packaged application in an image using Docker Hub.
```
Comment (aevesdocker) [inline get-started/publish-your-own-image.md:7]:
```suggestion
Follow this guide to learn how you can share your packaged application in an image using Docker Hub.
```

File: get-started/publish-your-own-image.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+## Step 1: Get an image
+
+Before you publish your image, you'll need an image to publish. For this guide, you'll use the `welcome-to-docker` image.
```
Comment (aevesdocker) [inline get-started/publish-your-own-image.md:11]:
```suggestion
Before you publish your image, you need an image to publish. For this guide, use the `welcome-to-docker` image.
```

File: get-started/publish-your-own-image.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+Before you can publish your image to Docker Hub, you need to rename it so that Docker Hub knows that the image is yours. Run the following `docker tag` command in your terminal to rename your image. Replace `YOUR-USERNAME` with your Docker ID.
+
+```console
+$ docker tag docker/welcome-to-docker YOUR-USERNAME/welcome-to-docker
+```
+
+## Step 4: Push your image to Docker Hub
+
+In Docker Desktop, go to the **Images** tab and find your image. In the **Actions** column, select the **More image actions** icon and then select **Push to Hub**. Your image will upload to Docker Hub and be publicly available for anyone to use.
```
Comment (aevesdocker) [inline get-started/publish-your-own-image.md:33]:
```suggestion
In Docker Desktop, go to the **Images** tab and find your image. In the **Actions** column, select the **More image actions** icon and then select **Push to Hub**. Your image is uploaded to Docker Hub and is publicly available for anyone to use.
```

File: get-started/publish-your-own-image.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+![Push image to Docker Hub](images/getting-started-push.gif)
+
+## Step 5: Verify the image is on Docker Hub
+
+That's it! Your image is now being shared on Docker Hub. In browser, go to [Docker Hub](https://hub.docker.com) and verify that you see the `welcome-to-docker` repository.
```
Comment (aevesdocker) [inline get-started/publish-your-own-image.md:39]:
```suggestion
That's it! Your image is now shared on Docker Hub. In your browser, go to [Docker Hub](https://hub.docker.com) and verify that you see the `welcome-to-docker` repository.
```

File: get-started/what-is-a-container.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+## Step 2: View the frontend
+
+The frontend is accessible on port 8088 of your local host. Select the link in the **Port(s)** column of your container, or visit http://localhost:8088  in your browser to check it out.
+
+![Accessing container frontend from Docker Desktop](images/getting-started-frontend.png){:width="500px"}
+
+## Step 3: Explore your container
+
+Docker Desktop allows you to easily view and interact with different aspects of your container. Try it out yourself, select your container to view logs, run commands in your container, and more.
```
[the remark this answers is not in the document]
Comment (craig-osterhout) [reply get-started/what-is-a-container.md]: From some other internal feedback, reworded this to focus on the Files tab and try to hammer in the concept of isolation.

**Review by aevesdocker — APPROVED** [review]
Looks great. Not sure about the change to 'hands-on guide' (not a blocker though). What's the difference between a hands-on guide and any other guide we have?

**Comment by craig-osterhout** [comment]
> Looks great. Not sure about the change to 'hands-on guide' (not a blocker though). What's the difference between a hands-on guide and any other guide we have?

There's no difference. There was a small concern from the team that the "quick start" title didn't match any of the wording in the learning center links to tie it all together.

**Comment by aevesdocker** [comment]
@craig-osterhout shouldn't this be merged with the DD 4.18 release on Thursday?

**Pull request merged**